rudder-core/src/main/resources/hooks.d/node-policy-generated/10-cf-promise-check.hook

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Hooks parameter are passed by environment variable: 
+#
+# - RUDDER_GENERATION_DATETIME    : generation datetime: ISO-8601 date/time that identify that policy generation start 
```

Comment:
By that you mean the `2016-10-28T09:45Z` format?
I did not get which exactly you meant after reading https://en.wikipedia.org/wiki/ISO_8601